README.md

```diff
@@ -30,3 +30,88 @@ The aims of the project are:
 3.	Calculate a ratio between public and private transport journey times 
 4.	Calculate an overall metric for the entire city
 5.	Compare the results for each city
+
+### Using VS code
+
+#### Create new branch 
+
+Initially created a branch on GitHub from the develop branch. 
+Within the terminal:
+```python 
```

Comment:
Love the codeblock here Chloe. However, this is not Python, this would be on the command line. I am not sure if that has a language name but I would guess it might be "shell" or "sh" or "bash". However just using the three backticks works too.